.kitchen.cloud.yml

```diff
@@ -44,6 +44,9 @@ platforms:
   driver_config:
     username: ubuntu
     image_ref: "Ubuntu 14.04"
+  run_list:
+   - recipe[apt]
+   - recipe[python-webapp-test::default]
```

Comment:
Why do you have all the ``recipe[python-webapp-test::default]`` included under each platform? Shouldn't that be covered by the default suite?